Makefile

```diff
@@ -161,7 +161,7 @@ dist: dist/toree pip-release
 dev: DOCKER_WORKDIR=/srv/toree/etc/examples/notebooks
 dev: SUSPEND=n
 dev: DEBUG_PORT=5005
-dev: .toree-dev-image dist
+dev: dist .toree-dev-image
```

Comment:
I think it's a bug, we should do `dist` first, so that the Dockerfile's `ADD` command could see the artifacts.

https://github.com/apache/incubator-toree/blob/937c1260bf7973ef20b0052989398ac9e789d160/Dockerfile.toree-dev#L52